### PR TITLE
Add favicon to makefile

### PR DESCRIPTION
**Pull request by Shard**
This adds an extra command for generating a 32x32 favicon.

### CODE REVIEW

**Comment by davidchambers** [comment]
I haven't created a favicon for a few years, so I don't know the current state of affairs. Do all modern browsers support PNG favicons, or do we need to provide an ICO file instead?

**Comment by Shard** [comment]
The current state of affairs is that IE11 added full proper support to pngs, before that IE had issues with PNG's. All other browsers have supported PNG for years as it's what the w3c spec defaults to.

It's possible we might want to make an ICO to cover IE10 down, although to be honest I haven't bothered with them for a few years now working as a front-end dev.

http://caniuse.com/#search=png%20favicon

**Comment by davidchambers** [comment]
Thanks for the information, @Shard. PNG sounds good to me. I did wonder why we'd provide a 32x32 PNG rather than reuse the 200x200 one, until I realized that a 32x32 PNG derived from an SVG will be crisper than a 200x200 PNG (derived from an SVG) shrunk to 32x32.

File: Makefile [hunk 1]
```diff
@@ -3,11 +3,14 @@ FONT = AvenirNext-Medium.ttf
 
 
 .PHONY: all
-all: sanctuary-logo.png sanctuary-logo.svg
+all: sanctuary-logo.png sanctuary-favicon.png sanctuary-logo.svg
```
Comment (davidchambers) [inline Makefile]: Would you mind switching `sanctuary-logo.png` and `sanctuary-favicon.png` above and in the target definitions below, to preserve alphabetical order? :smile:

**Comment by Shard** [comment]
I have reordered the favicon to be before the logo and squashed the changes.

**Comment by davidchambers** [comment]
:deciduous_tree:

**Pull request merged**